bundles/org.openhab.binding.meteofrance/src/main/java/org/openhab/binding/meteofrance/internal/handler/VigilanceHandler.java

```diff
@@ -0,0 +1,189 @@
+                updateDate(END_TIME, period.endValidityTime());
+            }
+            DomainId domainId = handler.requestMapData(domain, Term.TODAY);
+            ZonedDateTime now = ZonedDateTime.now().withZoneSameInstant(zoneId);
+            if (domainId != null) {
+                Map<Hazard, Risk> channels = new HashMap<>(Hazard.values().length);
+                Hazard.AS_SET.stream().filter(Hazard::isChannel).forEach(hazard -> channels.put(hazard, Risk.VERT));
+
+                domainId.phenomenonItems().forEach(phenomenon -> {
+                    Risk risk = phenomenon.phenomenonMaxColorId();
+                    for (TimelapsItem item : phenomenon.timelapsItems()) {
+                        if (item.contains(now)) {
+                            risk = item.colorId();
+                        }
+                    }
+                    channels.put(phenomenon.phenomenonId(), risk);
+                });
+
+                channels.forEach((k, v) -> updateAlert(k.channelName, v));
+
+                TextBlocItem bloc = handler.requestTextData(domain);
+                if (bloc != null) {
+                    String comment = bloc.blocItems().stream().filter(bi -> bi.type().equals(BlocType.SITUATION))
+                            .flatMap(bi -> bi.textItems().stream()).flatMap(ti -> ti.termItems().stream())
+                            .map(term -> term.getText(now)).collect(Collectors.joining("."));
+                    updateState(COMMENT, comment.isEmpty() ? UnDefType.NULL : StringType.valueOf(comment));
+                }
+                updateStatus(ThingStatus.ONLINE);
+            } else {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE, "No data available for the department");
+            }
+        }, () -> logger.warn("No viable bridge"));
+    }
+
+    private void updateAlert(String channelId, Risk value) {
+        State state = value != Risk.UNKNOWN ? new DecimalType(value.riskLevel) : UnDefType.NULL;
+        if (isLinked(channelId)) {
+            updateState(channelId, state);
+        }
+
+        String channelIcon = channelId + "-icon";
+        if (isLinked(channelIcon)) {
+            InputStream icon = iconProvider.getIcon(channelId, state.toString());
+            if (icon != null) {
+                try {
+                    State result = new RawType(icon.readAllBytes(), "image/svg+xml");
+                    updateState(channelIcon, result);
+                } catch (IOException e) {
+                    logger.warn("Error getting icon for channel {} and value {}: {}", channelId, value, e.getMessage());
+                }
+            } else {
+                logger.warn("Null icon returned for channel {} and state {}", channelIcon, state);
+            }
+        }
+    }
+
+    private void updateDate(String channelId, ZonedDateTime zonedDateTime) {
+        if (isLinked(channelId)) {
+            updateState(channelId, new DateTimeType(zonedDateTime));
```

Comment:
See earlier comment about respecting the openHAB configured timezone with the `TimeZoneProvider`

Reply:
Changed.